conceptual/Npgsql/toc.md

```diff
@@ -28,6 +28,7 @@
 # [Bulk Copy](copy.md)
 # [Logging](logging.md)
 # [Visual Studio Integration](ddex.md)
+# [FAQ](faq.md)
 # [API Reference](../../obj/api/Npgsql/)
```

Comment:
Should FAQ go after API reference?

Reply:
I'm not sure, I don't think so... FAQ is still conceptual documentation (articles in English), API documentation is a really different kind of docs... We should review the FAQ stuff too at some point, I think it's not very updated.